unraid_mcp/tools/info.py

```
@@ -0,0 +1,407 @@
+                return dict(data.get("upsDeviceById", {}))
+
+            if action == "ups_config":
+                return dict(data.get("upsConfiguration", {}))
+
+            raise ToolError(f"Unhandled action '{action}' — this is a bug")
+
+        except ToolError:
+            raise
+        except Exception as e:
+            logger.error(f"Error in unraid_info action={action}: {e}", exc_info=True)
+            raise ToolError(f"Failed to execute info/{action}: {str(e)}") from e
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)